File: 2020/6/1.js
// 2020 Day 6 Part 1
const fs = require('fs');
let input = fs.readFileSync('input.txt', 'utf-8')
  .split("\n\n")
  .map(x => {
    return x.replace(/\n/g, "")
      .split('')
      .filter(function(item, pos, self) {
        return self.indexOf(item) == pos;
      })
      .length;
  }).reduce((a, b) => a + b);

console.log(input);

File: 2017/02/1.swift
import Foundation

let input = try String(contentsOfFile: "./input.txt")
  .split(separator: "\n")
  .map { $0.split(separator: "\t").map { Int($0)! }.sorted(by: < ) }

print(input.map { $0.last! - $0.first! }.reduce(0, +))
